apf/apfNumbering.cc

```diff
@@ -20,6 +20,7 @@ enum { FIXED = -2, FREE_BUT_NOT_NUMBERED = -1 };
 
 template <class T>
 NumberingOf<T>::NumberingOf()
+  : FieldBase()
```

Comment:
This appears to be the only non-whitespace change.  Would you please remove the whitespace changes below?

Reply:
resolved